package NYT.NSecurityClient.NProto;

import "yt/ytlib/misc/guid.proto";
import "yt/ytlib/rpc/rpc.proto";

////////////////////////////////////////////////////////////////////////////////

message TAuthenticatedExt
{
    extend NRpc.NProto.TRequestHeader
    {
        optional string authenticated_user = 102;
    }
}

////////////////////////////////////////////////////////////////////////////////
